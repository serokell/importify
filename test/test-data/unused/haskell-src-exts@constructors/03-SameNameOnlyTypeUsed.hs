module SameNameOnlyTypeUsed where

import Language.Haskell.Exts (Module (Module))

foo :: Module ()
foo = undefined
