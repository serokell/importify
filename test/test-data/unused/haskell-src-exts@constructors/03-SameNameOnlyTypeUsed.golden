module SameNameOnlyTypeUsed where

import Language.Haskell.Exts (Module         )

foo :: Module ()
foo = undefined
